localsettings/extensions-DiscussionTools.txt

```diff
@@ -0,0 +1,2 @@
+$wgLocaltimezone = 'UTC';
```

Comment:
since these configuration settings are for settings in core, I would suggest putting them in the overall settings that are loaded each time, to have a more stable interface across different patchdemo wikis - the other configuration files are for extension settings

Reply:
Done